app/controllers/bill_controller.rb

```diff
@@ -0,0 +1,26 @@
+class BillController < ApplicationController
+
+  def index
+    @bills = Bill.all
+  end
+
+  def new
+    @bill = Bill.new
+  end
+
+  def create
+  end
+
+  def show
+    @bill = Bill.find(params[:id])
+  end
+
+  def edit
+    @bill = Bill.find(params[:id])
+  end
+
+  def update
```

Comment:
line 22: This is where a bill might get updated to paid? Not sure. where would this redirect to after it's completed? Where would we want to go after this? Do we even have to go anywhere? Maybe we just render the bill show page with the updated status of the bill? Would that be an html request to the merchant? Json to the user? Both simultaneously?